plugins/techdocs/src/reader/transformers/styles/rules/layout.ts

```diff
@@ -47,6 +47,11 @@ export default ({ theme, sidebar }: RuleOptions) => `
   height: 20px !important;
 }
 
+.md-nav__item--active > .md-nav__link, a.md-nav__link--active {
+  text-decoration: underline;
```

Comment:
Could we remove the "underline" style to have the same appearance as other active list items in Backstage?

Reply:
Let's keep it in favor of the color blind 🙂 :
https://github.com/backstage/backstage/pull/12679#issuecomment-1196385940